### PR TITLE
Add slider

### PR DESCRIPTION
**Pull request by VolodymyrButs**

### CODE REVIEW

File: src/components/slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    list-style: none;
+    text-align: center;
+}
+.react-multi-carousel-dot button {
+    display: inline-block;
+    width: 12px;
+    height: 12px;
+    border-radius: 50%;
+    opacity: 1;
+    padding: 5px 5px 5px 5px;
```
Comment (dkupchyk) [inline src/components/slider/slider.css]: Replace with padding: 5px;

File: src/components/slider/slider.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    left: 0;
+    right: 0;
+    justify-content: center;
+    margin: auto;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+    text-align: center;
+}
+.react-multi-carousel-dot button {
```
Comment (dkupchyk) [inline src/components/slider/slider.css]: There are two padding properties, remove one of them

File: src/components/slider/slider.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    opacity: 1;
+    padding: 5px 5px 5px 5px;
+    box-shadow: none;
+    transition: background 0.5s;
+    border-width: 2px;
+    border-style: solid;
+    border-color: grey;
+    padding: 0;
+    margin: 0;
+    margin-right: 6px;
```
Comment (dkupchyk) [inline src/components/slider/slider.css]: Suggest remove the line before and rewrite to margin: 0 6px 0 0;

Comment (VolodymyrButs) [reply src/components/slider/slider.css]: its default styles file from the Carousell package! I Will try to resolve all comments

**Review by dkupchyk — APPROVED** [review]

**Pull request merged**